frontend_tests/node_tests/rendered_markdown.js

```diff
@@ -196,14 +202,17 @@ run_test("timestamp-twenty-four-hour-time", () => {
     // We will temporarily change the 24h setting for this test.
     with_field(page_params, "twenty_four_hour_time", true, () => {
         rm.update_elements($content);
-        assert.equal($timestamp.html(), '<i class="fa fa-clock-o"></i>\nWed, Jul 15 2020, 20:40\n');
+        assert.equal(
+            $timestamp.html(),
+            '<i class="fa fa-clock-o"></i>\nWed, Jul 15 2020, 20:40 UTC\n',
+        );
     });
 
     with_field(page_params, "twenty_four_hour_time", false, () => {
         rm.update_elements($content);
         assert.equal(
             $timestamp.html(),
-            '<i class="fa fa-clock-o"></i>\nWed, Jul 15 2020, 8:40 PM\n',
+            '<i class="fa fa-clock-o"></i>\nWed, Jul 15 2020, 8:40 PM UTC\n',
         );
     });
 });
```

Comment:
Can you add another test here that sets a different base timezone, so that output shows something other than UTC?

Reply:
Thanks, Added tests within 1ab88f91401717af6310f1003155482434ce94b3